conda/lock-environment.sh

```diff
@@ -9,7 +9,7 @@ LOCK_ENV='CondaLock'
 conda env list | grep ${LOCK_ENV} > /dev/null
 
 if [[ $? -eq 1 ]]; then
-  conda create -q -y -n ${LOCK_ENV} -c conda-forge conda-lock=0.13 mamba=0.17
+  conda create -q -y -n ${LOCK_ENV} -c conda-forge python=3.9 conda-lock=0.13 mamba=0
```

Comment:
`Mamba` required as version `0`?

I would also remove the added `python` again since we are not depending on a specific version there and can let conda handle the appropriate Python version with the `conda-lock` and `mamba` being specified to a certain version